object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -279,6 +306,276 @@ private static String getExactStringValue(Tree tree, ValueAnnotatedTypeFactory f
     return null;
   }
 
+  @Override
+  public void postAnalyze(ControlFlowGraph cfg) {
+    checkCFG(cfg.getEntryBlock(), cfg);
+    super.postAnalyze(cfg);
+  }
+
+  private void checkCFG(SpecialBlock entry, ControlFlowGraph cfg) {
```

Comment:
Choose a more informative name, like `checkAlwaysCall`.  Also it would be good to write a Javadoc comment with an overview of what this method does.

Also I don't think `entry` needs to be a parameter to the method.  You could just call `cfg.getEntryBlock()` at the start of the method.